velox/core/QueryConfig.h

```diff
@@ -205,6 +205,10 @@ class QueryConfig {
   static constexpr const char* kHashProbeFinishEarlyOnEmptyBuild =
       "hash_probe_finish_early_on_empty_build";
 
+  /// The minimum table size in row that triggers the parallel join table build.
```

Comment:
What does "table size in row" mean?

Reply:
Change to "The number of table rows" to make it easier to understand.